cmd/upgrade/roles/cmd.go

```diff
@@ -842,6 +843,18 @@ func upgradeOperatorRolePoliciesFromCluster(
 		if err != nil {
 			return err
 		}
+
+		if operatorRoleName != "" {
+			err = awsClient.AttachRolePolicy(operatorRoleName, policyARN)
```

Comment:
If it is already attached does this error out?

Reply:
I don't believe it errors out. It (awsClient.AttachRolePolicy) is currently being done with account roles in "rosa upgrade roles".